prometheus-ingestion-retrieval.md

```diff
@@ -0,0 +1,67 @@
+# Prometheus Ingestion
+
+IRONdb has native endpoints for accepting remote write data from a Prometheus
+installation.  Once the Prometheus module is enabled, data can be send to IRONdb 
+by setting the Prometheus remote_write endpoint to:
+
+`http://irondbnode:8112/module/prometheus/write/<accountid>/<uuid>`
+
+## Enabling Prometheus Module
+
+IRONdb must be [configured](configuration.html) such that the Prometheus module is
+enabled for reading or writing Prometheus data natively.  In the <modules> section of 
+the IRONdb config, Prometheus support is activated by including the following stanza:
+```
+<generic image="prometheus" name="prometheus"/>
+```
+Depending on installation this may be shipped enabled by default.
```

Comment:
This is confusing to me.  It is not enabled anywhere by default today.  IRONdb standalone needs to add the above line to `/opt/circonus/etc/irondb-modules-site.conf` (there is a comment in that file explaining exactly how).  IMO that is all we need to document today.  Full Circonus Inside users are not supposed to write anything directly to IRONdb (they should write to a broker instead), so this is probably not even relevant for that type of deployment.

Reply:
I am likely misunderstanding Theo's comment that "that is shipped with irondb's default configs already".   I worded it this way for the future when we may have IRONdb public, but a paid version with modules installed as an offering, or whatever.   I'm open to better phrasing :)

Reply:
Changed the last sentence after discussion with Theo.